uetools/UeDatabase/Database.py

```diff
@@ -156,23 +145,30 @@ def create_database(self, path, database):
             print("===== CREATING NEW CASE DUMPS =====")
             for newdbfolder in tqdm(createdb):
                 subdir = newdbfolder.split("/")[-1]
-                self.chdir(path)
-                self.chdir(newdbfolder)
-                Case("input.yaml", verbose=False).save(
-                    "{}{}.hdf5".format(subdir, self.dbidentifier)
-                )
-                self.cases[
-                    "{}/{}".format(subdir, "{}{}".format(subdir, self.dbidentifier))
-                ] = Case(
-                    "{}{}.hdf5".format(subdir, self.dbidentifier),
-                    inplace=True,
-                    verbose=False,
+                identifier = f"{subdir}/{subdir}{self.dbidentifier}"
+
+                yaml_file = os.path.join(newdbfolder, "input.yaml")
+                hdf5_file = os.path.join(
+                    newdbfolder, f"{subdir}{self.dbidentifier}.hdf5"
                 )
-                self.chdir(path)
-        self.top()
+
+                # Load the case from YAML, save as HDF5
+                #
+                # Note: This modifies the global UEDGE state, so the
```

Comment:
Could this order-dependency be circumvented by utilizing the ParallalCase class you created? In my opinion that could be used for most database-related tasks and uses.